src/_P093_MitsubishiHP.ino

```diff
@@ -8,12 +8,14 @@
 /** Changelog:
  * 2023-05-04 tonhuisman: Add support for PLUGIN_GET_CONFIG_VALUE to enable fetching all available values (as included in the json)
  * 2023-05-04 tonhuisman: Start Changelog
+ * 2023-09-21 jfmennedy: Add support for "SetRemoteTemperature" Issue#4711
```

Comment:
This changelog is supposed to be from the top from new to old... 😉 I see why this one can be confusing, but the Start message should have been the giveaway 🤔

Reply:
modified ;-)